### PR TITLE
Sell out admin overview

### PR DESCRIPTION
**Pull request by prasannamaddireddy**

### CODE REVIEW

File: src/components/home/superUserApproverOverview.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+import { Container, Row, Col, Card, Button } from "react-bootstrap";
+import { useNavigate } from "react-router-dom";
+import calender from "./../../images/calender.png";
+import roles from "../../data/roles.json";
+
+function SuperUserApproverOverview(props) {
+  const navigate = useNavigate();
+
+  const partnerDataNavigation = () => {
+    navigate(`/partnerList?id=${roles.superUser}`);
```
Comment (nelson-b) [inline src/components/home/superUserApproverOverview.js:12]: chg to `/partnerList?role=

File: src/components/partnerList/partnerList.js [hunk 2]
```diff
@@ -34,7 +33,8 @@ function PartnerList(props) {
   };
 
   const handleRequest = () => {
-    navigate("/partnerRequestList");
+    navigate(`/partnerRequestList?id=${screenRole}`);
```
Comment (nelson-b) [inline src/components/partnerList/partnerList.js]: upd to /partnerRequestList?role=

File: src/components/partnerRequestList/partnerRequestList.js [hunk 3]
```diff
@@ -277,12 +281,17 @@ function PartnerRequestList({}) {
...
   }, []);
 
+  const handleCheckboxClick = (params) => {
+    const selectedRows = params.api.getSelectedRows();
+    setMessage(selectedRows?.length);
+  };
+
   const handleReject = () => {
-    setRowData(rowData);
+    alert(message ? `${message} Partner Accounts selected for Reject` : "");
```
Comment (nelson-b) [inline src/components/partnerRequestList/partnerRequestList.js]: remove alerts, just keep console log for now

File: src/components/partnerRequestList/partnerRequestList.js [hunk 4]
```diff
@@ -277,12 +281,17 @@ function PartnerRequestList({}) {
...
   };
 
   const handleApprove = () => {
-    setRowData(rowData);
+    alert(message ? `${message} Partner Accounts selected for Approval` : "");
```
Comment (nelson-b) [inline src/components/partnerRequestList/partnerRequestList.js]: remove alerts, just keep console log for now

File: src/components/partnerRequestList/partnerRequestList.js (hunk 3 above)
Comment (prasannamaddireddy) [reply src/components/partnerRequestList/partnerRequestList.js]: done with comments

**Review by nelson-b — APPROVED** [review]
fine, pls assign unique proper variable name

**Pull request merged**